src/core/inlines.js

```diff
@@ -99,16 +99,11 @@ export function run(conf) {
             df.appendChild(document.createTextNode("]"));
 
             if (illegal && !conf.normativeReferences.has(ref)) {
-              cite.classList.add("respec-offending-element");
-              const msg =
+              showInlineWarning(
+                cite,
                 "Normative references in informative sections are not allowed. " +
-                `Remove '!' from the start of the reference \`[[!${ref}]]\`. `;
-              pub(
-                "warn",
-                msg + "See developer console to find offending element."
+                  `Remove '!' from the start of the reference \`[[!${ref}]]\``
```

Comment:
😆 
``` html
<cite 
 class="respec-offending-element" 
 title="Normative references in informative sections are not allowed. Remove '!' from the start of the reference `[[!html]]`" 
 id="respec-offender-normative-references-in-informative-sections-are-not-allowed-remove-from-the-start-of-the-reference-html"
><a class="bibref" href="#bib-html">html</a></cite>
```
Should shorten it? But it maybe fine, as the `id` is there just for error linking.

Reply:
I think it should be just okay, as it won't be used as an external link.